NGitLab.Tests/Docker/GitLabDockerContainer.cs

```diff
@@ -26,7 +27,9 @@ public class GitLabDockerContainer
     public const string ImageName = "gitlab/gitlab-ee";
 
     // https://hub.docker.com/r/gitlab/gitlab-ee/tags/
-    public const string GitLabDockerVersion = "15.11.9-ee.0"; // Keep in sync with .github/workflows/ci.yml
+    public const string GitLabDockerVersion = "16.11.3-ee.0"; // Keep in sync with .github/workflows/ci.yml
+    // public const string GitLabDockerVersion = "16.10.6-ee.0"; // Keep in sync with .github/workflows/ci.yml
+    // public const string GitLabDockerVersion = "15.11.13-ee.0"; // Keep in sync with .github/workflows/ci.yml
```

Comment:
```suggestion
    public const string GitLabDockerVersion = "16.11.3-ee.0"; // Keep in sync with .github/workflows/ci.yml
```

I don't think it's really needed to let the examples here

Reply:
I apologize. I commited my test code by mistake ...

Reply:
Removed